{"name": "test_2_CreateLog", "status": "broken", "statusDetails": {"message": "TypeError: 'tuple' object is not callable", "trace": "self = <testcases.test_create_log.TestCreate_log object at 0x03D1C880>\ntest_logsystem = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"462f1675-ad06-43dd-ba98-021fef5d4e43\")>\n\n    def test_2_CreateLog(self,test_logsystem):\n        self.page1 = TestLogPage(test_logsystem)\n        self.page1.type_blog_title(\"我最帅\")\n        self.page1.type_tag(\"个人属性\")\n        self.page1.type_ke_content(\"我最帅，在座的各位谁赞成，谁反对？\")\n        time.sleep(2)\n>       self.page1.select_album()\nE       TypeError: 'tuple' object is not callable\n\ntest_create_log.py:33: TypeError"}, "start": 1628835264507, "stop": 1628835266545, "uuid": "d112198c-e4db-4fa5-b591-e0e554fce4cd", "historyId": "4cc768d0db640219c705049bf0abf532", "testCaseId": "b1dec58d31e6c5197e5bdc5714d1b0e5", "fullName": "testcases.test_create_log.TestCreate_log#test_2_CreateLog", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_create_log"}, {"name": "subSuite", "value": "TestCreate_log"}, {"name": "host", "value": "DESKTOP-S2QKEKE"}, {"name": "thread", "value": "10300-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_create_log"}]}